### PR TITLE
#24373 : Documentation/Navbar : Refer to utility classes used. Hidden when printing. 

### PR DESCRIPTION
**Pull request by aavmurphy**
refer to the  (not navbar) classes used for newbies
add: navbars don't print
give the 1st example a drop-down (as its the template people will copy and adapt)

### CODE REVIEW

File: docs/4.0/components/navbar.md [hunk 1]
```diff
@@ -29,7 +29,10 @@ Navbars come with built-in support for a handful of sub-components. Choose from
 - `.navbar-text` for adding vertically centered strings of text.
 - `.collapse.navbar-collapse` for grouping and hiding navbar contents by a parent breakpoint.
 
-Here's an example of all the sub-components included in a responsive light-themed navbar that automatically collapses at the `lg` (large) breakpoint.
+Here's an example of all the sub-components.
+- the navbar is light-themed
+- its responsive - it automatically collapses at the `lg` (large) breakpoint.
```
Comment (XhmikosR) [inline docs/4.0/components/navbar.md]: `It's`

**Review by mdo — CHANGES_REQUESTED** [review]
Rather than add an entire new section to this page, let's put this under the `How it works` list.

**Comment by aavmurphy** [comment]
@mdo please re-check. have added 1 more thing (fixed-top/bottom uses position:fixed), so doesn't reserve a gap on the screen. couldn't thing of a nice way to phrase it though. thanks

**Review by mdo — APPROVED** [review]
I pushed another commit to fine tune some things here around the language and formatting of the content—more links to utilities pages, better formatting of paragraphs and lists, etc.

Thanks for the back and forth!

**Comment by aavmurphy** [comment]
Thanks!

**Pull request merged**